src/components/scrollbar/Scrollbar.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {SScrollbar } from './styles/SScrollbar';
+
+interface IScrollbar{
+
+}
+
+export const Scrollbar: React.FC<IScrollbar> = ({...props}) => {
```

Comment:
@steiningermona this component should forward the ref to the styled one. In some cases it is necessary to bind the scroll event to trigger some ui feedback or other effects.

https://styled-components.com/docs/advanced#refs